spec/models/piece_spec.rb

```diff
@@ -1,4 +1,62 @@
+    @b_pawn_1 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'black', row_coordinate: 1, column_coordinate: 0)
+    @b_pawn_2 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'black', row_coordinate: 1, column_coordinate: 1)
+    @b_pawn_3 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'black', row_coordinate: 3, column_coordinate: 2)
+    @b_pawn_4 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'black', row_coordinate: 1, column_coordinate: 4)
+    @b_pawn_5 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'black', row_coordinate: 1, column_coordinate: 5)
+    @b_pawn_6 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'black', row_coordinate: 1, column_coordinate: 6)
+    @b_pawn_7 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'black', row_coordinate: 1, column_coordinate: 7)
+    @w_knight_1 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Knight', color: 'white', row_coordinate: 3, column_coordinate: 3)
+    @w_bishop_1 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Bishop', color: 'white', row_coordinate: 5, column_coordinate: 0)
+    @w_pawn_1 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'white', row_coordinate: 5, column_coordinate: 1)
+    @w_pawn_2 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'white', row_coordinate: 6, column_coordinate: 2)
+    @w_pawn_3 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'white', row_coordinate: 6, column_coordinate: 3)
+    @w_pawn_4 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'white', row_coordinate: 6, column_coordinate: 4)
+    @w_pawn_5 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'white', row_coordinate: 6, column_coordinate: 5)
+    @w_pawn_6 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'white', row_coordinate: 6, column_coordinate: 6)
+    @w_pawn_7 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Pawn', color: 'white', row_coordinate: 6, column_coordinate: 7)
+    @w_rook_1 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Rook', color: 'white', row_coordinate: 7, column_coordinate: 0)
+    @w_king = FactoryGirl.create(:piece, game_id: @game.id, type: 'King', color: 'white', row_coordinate: 7, column_coordinate: 3)
+    @w_queen = FactoryGirl.create(:piece, game_id: @game.id, type: 'Queen', color: 'white', row_coordinate: 7, column_coordinate: 4)
+    @w_bishop_2 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Bishop', color: 'white', row_coordinate: 7, column_coordinate: 5)
+    @w_knight_2 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Bishop', color: 'white', row_coordinate: 7, column_coordinate: 6)
+    @w_rook_2 = FactoryGirl.create(:piece, game_id: @game.id, type: 'Rook', color: 'white', row_coordinate: 7, column_coordinate: 7)
+  end
```

Comment:
I just merged `populate_game` into master. I recommend merging that into your branch. Then you'll have all the pieces created for the game & can use them in these tests.